v3/src/components/case-table/collection-table.tsx

```diff
@@ -190,6 +173,34 @@ export const CollectionTable = observer(function CollectionTable(props: IProps)
     }
   }, [collectionTableModel?.rows, collectionTableModel?.inputRowIndex, showInputRow])
 
+  const { handleSelectedCellChange, navigateToNextRow } = useSelectedCell(gridRef, columns, rows)
+
+  function handleCellKeyDown(args: TCellKeyDownArgs, event: CellKeyboardEvent) {
+    if (args.mode === "EDIT") {
+      if (blockAPIRequestsWhileEditing(data) && ["Enter", "Tab"].includes(event.key)) {
+        // Note that this doesn't account for the case in which the Tab key results in focus exiting
+        // the table altogether. If we need to account for that, we can replicate the logic from
+        // RDG's internal `canExitGrid()` function or perhaps add a callback to RDG that is called
+        // when focus exits the grid.
+        uiState.setIsNavigatingToNextEditCell(true)
+        setTimeout(() => uiState.setIsNavigatingToNextEditCell(false), 500)
```

Comment:
Yes, it didn't work. But thinking about it more this morning, we can probably get rid of `isNavigatingToNextEditCell` altogether now, since API requests now come in when a cell is being edited but hasn't been modified, which is the exact case we find ourselves in after pressing enter or tab.

Reply:
Getting rid of it would be a nice resolution.